src/Sitecore.Modules.WeBlog/Sitecore.Modules.WeBlog.csproj

```diff
@@ -54,14 +54,6 @@
       <HintPath>$(SitecorePath)\bin\HtmlAgilityPack.dll</HintPath>
       <Private>False</Private>
     </Reference>
-    <Reference Include="HtmlSanitizationLibrary, Version=4.3.0.0, Culture=neutral, PublicKeyToken=d127efab8a9c114f, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\AntiXSS.4.3.0\lib\net40\HtmlSanitizationLibrary.dll</HintPath>
-      <Private>True</Private>
-    </Reference>
-    <Reference Include="Lucene.Net">
-      <HintPath>$(SitecorePath)\bin\Lucene.Net.dll</HintPath>
-      <Private>False</Private>
-    </Reference>
```

Comment:
Are you talking about `HtmlSanitizationLibrary`? 
It is delivered with `AntiXssLibrary.xml` but is not used in a part of API used by us.
I just created build env without this DLL and I am able to display comments without any problems.